example.py

```diff
@@ -507,6 +507,7 @@ def _next_timestamp() -> datetime:
 extracted_file_file_facet1 = uco.observable.FacetFile(
     file_extension="jpg",
     file_name="IMG_0123.jpg",
+    file_path="file/Image/",
```

Comment:
This should end with the file basename (`file_name` parameter here).

Reply:
@ajnelson-nist Done!

Reply:
I don't think this made it into your update.

Reply:
Sorry for asking for this without thinking too much harder.  I think it's ultimately unhelpful to have `filePath` on this node.